WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsForm.swift

```diff
@@ -58,72 +59,104 @@ struct WooShippingCustomsForm: View {
 
     var body: some View {
         NavigationView {
-        GeometryReader { geometry in
-            ScrollViewReader { proxy in
-                ScrollView {
-                        VStack(alignment: .leading, spacing: Constants.defaultVerticalSpacing) {
-                            HStack {
-                                Text(Localization.contentType)
-                                    .font(.subheadline)
-                                Spacer()
-                            }
+            GeometryReader { geometry in
+                VStack {
+                    ScrollViewReader { proxy in
+                        ScrollView {
+                            VStack(alignment: .leading, spacing: Constants.defaultVerticalSpacing) {
```

Comment:
I'm seeing an issue with too much vertical spacing between the labels and their corresponding dropdown menu or text field in this `VStack`, compared to the designs:

Design|Actual
-|-
<img width="231" alt="Screenshot 2024-12-30 at 16 23 46" src="https://github.com/user-attachments/assets/927525d1-2c50-45ca-959f-295d3811b8e5" />|<img width="338" alt="Screenshot 2024-12-30 at 16 23 35" src="https://github.com/user-attachments/assets/93f85041-212b-4df0-8b0b-c0b470cc9859" />

Reply:
That's true, I fixed it in 78df820bbe9c1ff59848a437d2386e959391bab3